public/classroom_poc.html

```
@@ -0,0 +1,733 @@
+        <div class="flex items-center gap-3">
+            <div class="flex items-center gap-2 px-3 py-1 bg-gray-800 rounded-full text-xs font-medium">
+                <span id="connDot" class="w-2 h-2 rounded-full bg-red-500 dot-pulse"></span>
+                <span id="connText" class="text-gray-400">Connecting…</span>
+            </div>
+            <div class="hidden sm:flex items-center gap-1.5 px-3 py-1 bg-gray-800 rounded-full text-xs font-medium text-gray-300">
+                <i data-lucide="hash" class="w-3 h-3 text-teal-400"></i>
+                <span id="topRoomId">—</span>
+            </div>
+        </div>
+        <div class="flex items-center gap-2">
+            <button id="btnParticipants"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)